subjects/mobile-dev/chess/audit/README.md

```diff
@@ -1,19 +1,51 @@
 #### Functional
 
-#### In order to run and hot reload app either on emulator or device, follow the [instructions](https://docs.flutter.dev/get-started/test-drive?tab=androidstudio#run-the-app).
+#### In order to run and hot reload the app either on emulator or device, follow the [instructions](https://docs.flutter.dev/get-started/test-drive?tab=androidstudio#run-the-app).
```

Comment:
This paragraph is a `>` section in the other audits

Reply:
done

Reply:
done